### PR TITLE
Update sqs-example.md

### PR DESCRIPTION
**Pull request by tsahiduek**
updating SQS example with the following changes:
1. Upgrading controller version to the latest one
2. Provide more details about the region parameter for the controller installation, and detail on how to deploy to other regions as stated in the other doc

Issue #, if available:

Description of changes:

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/content/docs/tutorials/sqs-example.md [hunk 1]
```diff
@@ -44,15 +44,18 @@ This guide assumes that you have:
...
+
 Log into the Helm registry that stores the ACK charts:
 ```bash
-aws ecr-public get-login-password --region us-east-1 | helm registry login --username AWS --password-stdin public.ecr.aws
+CONTROLLER_REGION=us-east-1
+aws ecr-public get-login-password --region $CONTROLLER_REGION | helm registry login --username AWS --password-stdin public.ecr.aws
 ```
 
-Deploy the ACK service controller for Amazon SQS using the [sqs-chart Helm chart](https://gallery.ecr.aws/aws-controllers-k8s/sqs-chart). Resources should be created in the `us-east-1` region:
+Deploy the ACK service controller for Amazon SQS using the [sqs-chart Helm chart](https://gallery.ecr.aws/aws-controllers-k8s/sqs-chart). We're defaulting the controller to deploy resources in the `us-east-1` region. If you're looking to deploy resources to other regions, please refer to the [Manage Resources In Multiple Regions]([url](https://aws-controllers-k8s.github.io/community/docs/user-docs/multi-region-resource-management/)) documentation.
```
Comment (Emil-Hansen) [inline docs/content/docs/tutorials/sqs-example.md:55]:
```suggestion
Deploy the ACK service controller for Amazon SQS using the [sqs-chart Helm chart](https://gallery.ecr.aws/aws-controllers-k8s/sqs-chart). If you're looking to deploy resources to multiple regions, please refer to the [Manage Resources In Multiple Regions]([url](https://aws-controllers-k8s.github.io/community/docs/user-docs/multi-region-resource-management/)) documentation.
```

File: docs/content/docs/tutorials/sqs-example.md [hunk 2]
```diff
@@ -44,15 +44,18 @@ This guide assumes that you have:
...
 
 ```bash
-helm install --create-namespace -n ack-system oci://public.ecr.aws/aws-controllers-k8s/sqs-chart --version=v0.0.3 --generate-name --set=aws.region=us-east-1
+helm install --create-namespace -n ack-system oci://public.ecr.aws/aws-controllers-k8s/sqs-chart --version=v1.0.4 --generate-name --set=aws.region=$CONTROLLER_REGION
```
Comment (Emil-Hansen) [inline docs/content/docs/tutorials/sqs-example.md:58]: Just checking this works, I wasn't able to install this version when I tried it before, I did manage to install 1.0.0

**Review by Emil-Hansen — APPROVED** [review]

File: docs/content/docs/tutorials/sqs-example.md (hunk 2 above)
Comment (tsahiduek) [reply docs/content/docs/tutorials/sqs-example.md]: You're right. It should be `1.0.4` (without the letter `v`)
Just tested it and it worked.
thanks

**Review by A-Hilaly — APPROVED** [review]
:+1:

**Comment by tsahiduek** [comment]
@A-Hilaly , thanks for approving this. 
It still seems like it needs the /lgtm label.

File: docs/content/docs/tutorials/sqs-example.md [hunk 3]
```diff
@@ -44,15 +44,18 @@ This guide assumes that you have:
 
 ### Install the ACK service controller for SQS
 
+> **_NOTE:_** This guide assumes you're using `us-east-1` as the region where the ACK controller will be deployed, as well as the Amazon SQS resource. If you want to create the object in another resource, simply change the region name to your region of choice.
+
 Log into the Helm registry that stores the ACK charts:
 ```bash
-aws ecr-public get-login-password --region us-east-1 | helm registry login --username AWS --password-stdin public.ecr.aws
+CONTROLLER_REGION=us-east-1
+aws ecr-public get-login-password --region $CONTROLLER_REGION | helm registry login --username AWS --password-stdin public.ecr.aws
```
Comment (RedbackThomson) [inline docs/content/docs/tutorials/sqs-example.md:52]: The ECR public endpoint is only available in us-east-1

Comment (tsahiduek) [reply docs/content/docs/tutorials/sqs-example.md]: Thank you. Just fixed that

**Comment by A-Hilaly** [comment]
Thank you @tsahiduek !
/lgtm

**Pull request merged**